boards/arm/twr_kv58f220m/doc/index.rst

```diff
@@ -0,0 +1,201 @@
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | soc flash                           |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SENSOR    | off-chip   | fxos8700 polling;                   |
+|           |            | fxos8700 trigger                    |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc                                 |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/arm/twr_kv58f220m/twr_kv58f220m_defconfig``.
+
+Other hardware features are not currently supported by the port.
+
+System Clock
+============
+
+The KV58 SoC is configured to use the 50 MHz external oscillator on the
+board with the on-chip PLL to generate a 237.5 MHz system clock.
+
+Serial Port
+===========
+
+The KV58 SoC has six UARTs. UART0 is configured for the console. The
+remaining UARTs are not used.
+
+Accelerometer and magnetometer
+==============================
+
+The TWR-KV58F220M board by default only supports polling the FXOS8700
+accelerometer and magnetometer for sensor values
+(``CONFIG_FXOS8700_TRIGGER_NONE=y``).
+
+In order to support FXOS8700 triggers (interrupts), shunts must be placed on
+the jumpers ``J2`` and ``J9``. A trigger option also must be enabled in Kconfig
+(either ``FXOS8700_TRIGGER_GLOBAL_THREAD=y`` or
+``FXOS8700_TRIGGER_OWN_THREAD=y``).
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+This board integrates an OpenSDA debug probe. However, it can currently only be
+used for flashing the KV58 SoC by copying the compiled firmware to the USB Mass
+Storage Device. The board cannot be debugged using the OpenSDA probe, since
+pyOCD does not support the target. The OpenSDA J-Link firmware also cannot be
```

Comment:
Can you reference the firmware version number here so the reader might notice when a newer (potentially working) version of the firmware becomes available.

Reply:
ack